<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>882</width>
    <height>548</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>lab1-3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mass:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edtYPlate"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="edtXPaper"/>
    </item>
    <item row="13" column="3">
     <widget class="QPushButton" name="btnWriteFile">
      <property name="text">
       <string>serialise</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="btnIsBalance">
      <property name="text">
       <string>is balance</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>l:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="edtHPaper">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>radius:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edtHPlate">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="edtLPaper"/>
    </item>
    <item row="1" column="2" rowspan="14">
     <widget class="QTextEdit" name="txtStack">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>w:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>h:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edtXPlate"/>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="edtWPaper">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="edtMassPaper"/>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="btnAddPaper">
      <property name="text">
       <string>add paper</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>h:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edtRPlate"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="edtYPaper"/>
    </item>
    <item row="14" column="3">
     <widget class="QPushButton" name="btnReadFile">
      <property name="text">
       <string>create from file</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="btnAddPlate">
      <property name="text">
       <string>add plate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnDelTop">
      <property name="text">
       <string>del top</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>clear stack</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edtMassPlate"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mass:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_1">
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Создать стопку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
